resources/us_verifications/models/us_components.yml

```diff
@@ -110,14 +110,14 @@ properties:
     $ref: "../../../shared/attributes/address/state.yml"
 
   zip_code:
-    type: string
-    description: The 5-digit ZIP code
-    pattern: ^\d{5}$
+    allOf:
+      - description: The 5-digit ZIP code
+      - $ref: "../../../shared/attributes/address/zip_code.yml"
```

Comment:
So this shared attribute actually includes the 9 digit zip code as a possibility (the pattern is `^\d{5}((-)?\d{4})?$`), but the descriptions here imply that the 5 digit zip and 4 digit add-on are separated.

I think it makes most sense to keep the original configuration intact for `zip_code` here, instead of referencing this attribute. Changing the attribute itself is out of the question, since it's referenced in multiple other models.